x/datapool/README.md

```diff
@@ -0,0 +1,34 @@
+## Deploy and Register NFT smart contract
+
+```shell
+TX_FLAG=(--chain-id {your chainID} --gas auto --gas-prices 5umed --gas-adjustment 1.3)
+MINTER=$(panacead keys show {your address or key of minter} -a)
+panacead tx datapool register-contract cw721_base.wasm --from $MINTER $TX_FLAG -y
```

Comment:
If I understand it correctly, the minter is the module account. Then, the name `MINTER` is not proper in this example. It would be reasonable to name it as `SENDER` or so.

Reply:
Yes it would be more proper name. changed it in [abecf64](https://github.com/medibloc/panacea-core/pull/288/commits/abecf64bd84313eb2a8de05373eb08f76fe860c8).